libnavigation-ui/src/main/res/values-night/colors.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <!-- Default SpeedLimitView -->
+  <color name="mapbox_speed_limit_text_color">#333333</color>
+  <color name="mapbox_speed_limit_view_background">#EFEFEF</color>
```

Comment:
Why are hardcoding the value here but not in "day" / regular `colors.xml`? https://github.com/mapbox/mapbox-navigation-android/blob/f8aca13f7140ac279e1d86ffca33d7b718416157/libnavigation-ui/src/main/res/values/colors.xml#L83